contracts/src/SP1Helios.sol

```diff
@@ -25,11 +27,14 @@ contract SP1Helios is AccessControlEnumerable {
     uint256 public immutable SLOTS_PER_EPOCH;
 
     /// @notice Role for updater operations
-    bytes32 public constant UPDATER_ROLE = keccak256("UPDATER_ROLE");
+    bytes32 public constant STATE_UPDATER_ROLE = keccak256("STATE_UPDATER_ROLE");
 
     /// @notice Role for updating VKEY
     bytes32 public constant VKEY_UPDATER_ROLE = keccak256("VKEY_UPDATER_ROLE");
 
+    /// @notice Role for adding / removing updaters
+    bytes32 public constant UPDATERS_ADMIN_ROLE = keccak256("UPDATERS_ADMIN_ROLE");
```

Comment:
Could we use `DEFAULT_ADMIN_ROLE` from `AccessControl` instead? This would mean all roles default to this role being the admin.